portfolio/src/main/webapp/courses.html

```diff
@@ -25,6 +26,11 @@
           <a class="nav-link" href="#">COURSES</a>
         </li>
       </ul>
+      <div class="icon-bar">
+        <a href="https://github.com/kpancha" class="github"><i class="fa fa-github"></i></a>
+        <a href="mailto:kirakpancha@gmail.com" class="email"><i class="fa fa-envelope"></i></a>
+        <a href="https://www.linkedin.com/in/kira-pancha-7ab886190/" class="linkedin"><i class="fa fa-linkedin"></i></a>
+      </div>
```

Comment:
Lets add a TODO here as well to extract this common icon bar html so we can fetch it and include it on various pages when we get to the server portions

Reply:
Would we be able to fetch it as part of the nav bar (rather than separately)?